tests/integration/targets/rds_param_group/tasks/main.yml

```diff
@@ -123,6 +123,25 @@
       - result.tags["Test"] == '123'
 
     # ============================================================
+
+  - name: test modifying rds parameter group engine/family (warning displayed)
+    rds_param_group:
+      name: '{{ rds_param_group.name }}'
+      engine: '{{ rds_param_group.engine_to_modify_to }}'
+      description: '{{ rds_param_group.description }}'
+      state: present
+      tags:
+        Environment: test
+        Test: 123
+    register: result
+
+  - name: verify that modifying rds param group engine/family failed
```

Comment:
You should update this task's name. Once done, LGTM!

Reply:
Fixed.